frames/MainWindow.py

```diff
@@ -17,6 +22,18 @@ class MainWindow(tk.Tk):
     def __init__(self):
         super().__init__()
 
+        ############################
+        # Setup Elastic Search
+        ############################
+        es = Elasticsearch([{'host': 'localhost', 'port': 9200}])
+        monsterFile = open('./json/data.json')
+        monsters = json.load(monsterFile)
+        print(monsters)
+        for monster in monsters:
+            es.index(index='monster_index', body = monster)
```

Comment:
I think you only have to add a document to an index once, so I don't think we need to do es.index() every time we run the program